scripts/mysqld_safe.sh

```diff
@@ -21,9 +21,11 @@ mysqld_ld_library_path=
 flush_caches=0
 numa_interleave=0
 wsrep_on=0
+simulate=0
```

Comment:
Its used in the mariadb-service-convert script https://github.com/MariaDB/server/pull/83/files#diff-80b1943e0e77f82475b779f31aeb83d7R12 to grab all the parsing work that mysqld_safe has done. I was hoping mysqld_safe.sh is something that would eventually die away, especially with systemd and similar innovations in other OSs.